crypto/fipsmodule/aes/asm/aesni-xts-avx512.pl

```diff
@@ -3126,125 +3038,6 @@
 ___
 }
 
-# Bits 7 & 4 contain the src1 register's MSB in inverted form
```

Comment:
Why was this perl handling removed? I believe that the replacement of the `.byte` instructions with `vaesenc|dec|enclast|declast|` and `vpclmulqdq` in the Perl output is causing the failures in gcc 7 and below.

Reply:
Yeah, I was hoping this was vestigial! Before I removed the copy code I did a bunch of clean up, for this bit I was really just crossing my fingers that it wasn't needed. I'll add it back in shortly.

Reply:
@pittma , you're still planning on adding it in, right?